src/components/modals/LoginModal.js

```diff
@@ -0,0 +1,126 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+
+import { ReactComponent as ExitLogo } from '../icons/exit.svg'
+import { ReactComponent as Eye } from '../icons/eye.svg'
+import { ReactComponent as EyeNo } from '../icons/eye_no.svg'
+
+import styles from './Modal.module.css'
+
+const LoginModal = ({
+    setIsLoginModalVisible,
+    setIsRegistrationModalVisible,
+    setToken,
+}) => {
+    const [isPaswordShow, setIsPaswordShow] = useState(false)
+    const [loginError, setLoginError] = useState('')
+    const [email, setEmail] = useState('')
+    const [pass, setPass] = useState('')
+    const [isLoading, setIsLoading] = useState(false)
+    const formData = {
+        identifier: email,
+        password: pass,
+    }
+
+    const login = e => {
+        e.preventDefault()
+        fetch('https://radiant-temple-07706.herokuapp.com/auth/local', {
+            method: 'POST',
+            headers: {
+                'Content-Type': 'application/json',
+            },
+            body: JSON.stringify(formData),
+        })
+            .then(data => data.json())
+            .then(body => {
+                if (body.jwt) {
+                    setToken(body.jwt)
+                    setIsLoading(false)
+                    setIsLoginModalVisible(false)
+                } else {
+                    setLoginError('Wrong credential')
+                    setIsLoading(false)
+                }
+            })
+    }
+    return (
+        <div className={styles.wrapper}>
+            <div className={styles.header}>
+                <div className={styles.containerHeader}>
+                    <button
+                        className={styles.closeButton}
+                        onClick={() => setIsLoginModalVisible(false)}
+                    >
+                        <ExitLogo className={styles.svg} />
+                    </button>
+                    <p>Вход</p>
+                    <span></span>
```

Comment:
? :)

Reply:
It's task for you) why I use this block?